src/app/configuration-manager/configuration-manager.component.html

```diff
@@ -27,19 +26,19 @@
                 </div>
               </div>
             </div>
-            <div class ="columns-fields">
+            <div class="columns-fields">
               <div class="field">
                 <div class="control">
-                  <input id="{{v.key}}" class="input" type="text" trim="blur" [value]="v?.value?.value != undefined ? v?.value?.value:''" [textContent]='v?.value?.value' (input)="onTextChange(v.key, true)"
-                  />
+                  <input id="{{category.key + '-' + v.key}}" class="input" type="text" trim="blur" [value]="v?.value?.value != undefined ? v?.value?.value:''"
```

Comment:
`category_name.lower().trim() + '-'+ config_item.lower().trim()`
if we can hold into a var and use, that will be great! it will be easy to debug / log in ts file too